x/evm/keeper/keeper_test.go

```diff
@@ -32,27 +32,31 @@ var (
 type KeeperTestSuite struct {
 	suite.Suite
 
-	ctx     sdk.Context
-	querier sdk.Querier
-	app     *app.EthermintApp
-	address ethcmn.Address
+	ctx         sdk.Context
+	app         *app.EthermintApp
+	queryClient types.QueryClient
+	address     ethcmn.Address
 }
 
 func (suite *KeeperTestSuite) SetupTest() {
 	checkTx := false
 
 	suite.app = app.Setup(checkTx)
-	suite.ctx = suite.app.BaseApp.NewContext(checkTx, abci.Header{Height: 1, ChainID: "ethermint-3", Time: time.Now().UTC()})
-	suite.querier = keeper.NewQuerier(*suite.app.EvmKeeper)
+	suite.ctx = suite.app.BaseApp.NewContext(checkTx, tmproto.Header{Height: 1, ChainID: "ethermint-3", Time: time.Now().UTC()})
```

Comment:
is tmproto.Header the same as abci.Header? can't find the old definition of abci.Header and was wondering if it should have proto encoding or is there the possibility that you would use amino to encode the header?